google/cloud/bigtable/client.py

```diff
@@ -576,17 +575,42 @@ async def sample_keys(
         """
         raise NotImplementedError
 
-    def mutations_batcher(self, **kwargs) -> MutationsBatcher:
+    def mutations_batcher(
+        self,
+        *,
+        flush_interval: float | None = 5,
+        flush_limit_mutation_count: int | None = 1000,
+        flush_limit_bytes: int = 20 * MB_SIZE,
+        flow_control_max_count: int | None = 100_000,
```

Comment:
should this flow_control_mutation_count?

Reply:
Good point, fixed